myTSReduxApp/store/pageReducer/select.ts

```diff
@@ -0,0 +1,14 @@
+import {createSelector} from 'reselect';
+
+import {State} from '../type';
+
+import {PageReducerState} from './reducer';
+
+export function selectPageReducerState(state: State): PageReducerState {
+  return state.pageReducer;
+}
```

Comment:
And here is that selector I was mentioning in my previous comment. Very clean encapsulation.